lib/registries/addon/edit-revision/-components/right-nav/template.hbs

```diff
@@ -78,3 +78,13 @@
         @date={{@revisionManager.revision.dateModified}}
     />
 </span>
+
+{{#if @revisionManager.showDeleteButton}}
+    <DeleteButton
+        local-class='DeleteButton'
+        @delete={{perform @onDelete}}
```

Comment:
You have access to `@revisionManager`. Would it be a good idea to add the `delete` task to eliminate the need for an extra parameter to be passed in to right-nav?

Reply:
This feels much more appropriate to live on the revisionManager. I will do that now!